packages/next/src/server/lib/incremental-cache/file-system-cache.ts

```diff
@@ -339,7 +339,13 @@ export default class FileSystemCache implements CacheHandler {
 
       await this.fs.writeFile(
         this.getFilePath(
-          `${key}.${isAppPath ? 'rsc' : 'json'}`,
+          `${key}.${
+            isAppPath
+              ? this.experimental.ppr
+                ? RSC_PREFETCH_SUFFIX
+                : RSC_SUFFIX
+              : 'json'
+          }`,
```

Comment:
This handles when we update the static RSC prefetch payload. Previously it just wrote a `.rsc` file when during PPR we instead have to write a `.prefetch.rsc` payload.